superset/db_engine_specs.py

```diff
@@ -1005,46 +1005,46 @@ def convert_to_hive_type(col_type):
             }
             return tableschema_to_hive_types.get(col_type, 'STRING')
 
+        bucket_path = config['CSV_TO_HIVE_UPLOAD_S3_BUCKET']
+
+        if not bucket_path:
+            logging.info('No upload bucket specified')
+            raise Exception(
+                'No upload bucket specified. You can specify one in the config file.')
+
         table_name = form.name.data
         schema_name = form.schema.data
+        full_table_name = ''
```

Comment:
Is this needed given it’s guaranteed to be defined in the if/else block and is scoped for the whole of the function.